subt_ign/launch/cloudsim_bridge.ign

```diff
@@ -10,6 +10,7 @@
     Usage: ign launch cloudsim_bridge.ign
             [worldName:=<worldName>
              robotNameX:=<robotName> robotConfigX:=<robotConfig>
+             marsupial:=true
```

Comment:
@nkoenig, will cloudsim need to be updated to accept this new arg?

Reply:
Yup, it's in a [MR](https://gitlab.com/ignitionrobotics/web/cloudsim/-/merge_requests/27)